python/mlc_chat/quantization/quantization.py

```diff
@@ -162,40 +163,42 @@ def quantize_weight(self, weight: tvm.runtime.NDArray) -> List[tvm.runtime.NDArr
         model_dtype="float16",
         linear_weight_layout="NK",
     ),
-    "fp8_e4m3_e5m2_max_calibration": GroupQuantize(
+    "fp8_e4m3_e5m2_max_calibration": PerTensorQuantize(
         name="fp8_e4m3_e5m2_max_calibration",
-        kind="group-quant",
-        group_size=64,
-        quantize_dtype="e5m2_float8",
+        kind="per-tensor-quant",
+        activation_dtype="e4m3_float8",
+        weight_dtype="e5m2_float8",
         storage_dtype="uint32",
         model_dtype="float16",
-        linear_weight_layout="NK",
+        no_scale=True,
+        quantize_embedding=False,
```

Comment:
What was the issue with take? Do we need different scheduling / custom kernel due to perf regression?

Reply:
`FuseDequantTake` expects dequantization being a single block, that means we need to fuse https://github.com/octoml/mlc-llm/pull/232/files#diff-48e1e238dcdc972bef819418bc1a6761afe31a7799d1054974a17759a23eb9dcR263-R274
into one block